tests/E2E/test/campaigns/full/08_stocks/4_update_product_quantity.js

```diff
@@ -157,11 +108,7 @@ scenario('Update quantity of a product', () => {
         .then(() => client.checkTextValue(Stock.success_hidden_panel,'Stock successfully updated', 'contain'));
     });
     for (let i = 1; i <= 3; i++) {
-      if (i === 1) {
```

Comment:
don't you want to `continue` on the first row?

Reply:
It continue, but only the first one was failing ^^